### PR TITLE
fix(chips): Add an event typedef for chip interaction events

### PR DESCRIPTION
**Pull request by rlfriedman**

### CODE REVIEW

File: packages/mdc-chips/chip-set/adapter.js [hunk 1]
```diff
@@ -15,7 +15,8 @@
  * limitations under the License.
  */
 
-/* eslint no-unused-vars: [2, {"args": "none"}] */
+/* eslint-disable no-unused-vars */
+import {MDCChipInteractionEventType} from '../chip/foundation';
```
Comment (bonniezhou) [inline packages/mdc-chips/chip-set/adapter.js]: I think we want to keep the `no-unused-vars: [2, {"args": "none"}]` rule for the rest of the file, so you can do something like this:
```
// eslint-disable-next-line no-unused-vars
import {MDCChipInteractionEventType} from '../chip/foundation';

/* eslint no-unused-vars: [2, {"args": "none"}] */
...
```

File: packages/mdc-chips/chip-set/foundation.js [hunk 2]
```diff
@@ -17,7 +17,8 @@
 
 import MDCFoundation from '@material/base/foundation';
 import MDCChipSetAdapter from './adapter';
-import MDCChipFoundation from '../chip/foundation';
+/* eslint-disable no-unused-vars */
```
Comment (bonniezhou) [inline packages/mdc-chips/chip-set/foundation.js]: Same here, use `// eslint-disable-next-line no-unused-vars`

File: packages/mdc-chips/chip-set/adapter.js (hunk 1 above)
Comment (rlfriedman) [reply packages/mdc-chips/chip-set/adapter.js]: Cool, done!

File: packages/mdc-chips/chip-set/foundation.js (hunk 2 above)
Comment (rlfriedman) [reply packages/mdc-chips/chip-set/foundation.js]: Done

**Pull request merged**